### ISSUE TITLE
NumberFormatException when executing /economy commands

### ISSUE THREAD

**Issue by paulhlschr** [lokka30/PhantomEconomy#27]
```
[21:59:28] [Server thread/INFO]: PaulLink issued server command: /eco set PaulLink 100
[21:59:28] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'eco' in plugin PhantomEconomy v1.9-RELEASE
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.15.2.jar:git-Paper-161]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.15.2.jar:git-Paper-161]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.dispatchCommand(CraftServer.java:752) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.PlayerConnection.handleCommand(PlayerConnection.java:1825) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1633) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.PacketPlayInChat.a(PacketPlayInChat.java:47) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:1038) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1031) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1015) ~[patched_1.15.2.jar:git-Paper-161]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:938) ~[patched_1.15.2.jar:git-Paper-161]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_241]
Caused by: java.lang.NumberFormatException: For input string: "50,00"
	at sun.misc.FloatingDecimal.readJavaFormatString(Unknown Source) ~[?:1.8.0_241]
	at sun.misc.FloatingDecimal.parseDouble(Unknown Source) ~[?:1.8.0_241]
	at java.lang.Double.parseDouble(Unknown Source) ~[?:1.8.0_241]
	at io.github.lokka30.phantomeconomy.utils.Utils.round(Utils.java:29) ~[?:?]
	at io.github.lokka30.phantomeconomy.VaultImplementer.withdrawPlayer(VaultImplementer.java:226) ~[?:?]
	at io.github.lokka30.phantomeconomy.commands.EconomyCommand.onCommand(EconomyCommand.java:136) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.15.2.jar:git-Paper-161]
	... 17 more
```

I tried to fix it by myself, but tbh I don't know, where this error comes from, as you haven't changed anything in the related classes. Occurs with Release 1.9.

**Comment by lokka30** [comment]
Another user had sent me this exact same issue and I can't find where this is from either.

**Comment by lokka30** [comment]
I'll take a guess that an error from a while ago where default balances weren't saved properly is affecting that account. Try:
* Stop the server
* Open `PhantomEconomy/data.json/`
* Set `players.YOUR_UUID.balance` to `50.00`
* Start the server

If the previous balance was `50,00` instead of `50.00` (comma vs period), then we've found the culprit. Otherwise... well, I've looked through the code and I can't find anything about this, so I couldn't have a clue.

**Comment by lokka30** [comment]
Confirmed, this is still a bug. Not sure why there is a comma in the format ???

**Comment by lokka30** [comment]
Pushed an unreleased version to the repo but haven't updated the plugin. Awaiting results from a user experiencing the issue first.
(Attempted to fix by using a different method to round)

**Comment by lokka30** [comment]
Fixed in v1.9.13-RELEASE
